src/code/z_actor_dlftbls.c

```diff
@@ -36,15 +36,15 @@ ActorOverlay gActorOverlayTable[] = {
 #undef DEFINE_ACTOR_INTERNAL
 #undef DEFINE_ACTOR_UNSET
 
-ActorId gMaxActorId = 0;
+s32 gMaxActorId = 0;
 
-FaultClient D_801ED930;
-FaultAddrConvClient D_801ED940;
+FaultClient sActorOverlayTableFaultClient;
+FaultAddrConvClient sActorOverlayTableFaultAddrConvClient;
 
-void ActorOverlayTable_FaultPrint(void* arg0, void* arg1) {
+void ActorOverlayTable_FaultClient(void* arg0, void* arg1) {
     ActorOverlay* overlayEntry;
     u32 overlaySize;
-    ActorId i;
+    s32 i;
```

Comment:
Yeah looking at it again I agree keeping ActorId as the type is better. I've also renamed the variables to `actorId`.

Reply:
only reason it would be preferred is sync with oot, since we dont use enums as types over there. 
idk if we'll ever come to an agreement on this topic, but the main argument for not doing so is that you cant consistently use enums as types everywhere. only where they happened to use s32 in their original code